packages/block-editor/src/components/use-display-block-controls/index.native.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { hasBlockSupport } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { useBlockEditContext } from '../block-edit/context';
+import { store as blockEditorStore } from '../../store';
+
+export default function useDisplayBlockControls() {
+	const { isSelected, clientId, name } = useBlockEditContext();
+	return useSelect(
+		( select ) => {
+			const { getBlockName, getBlockRootClientId } = select(
+				blockEditorStore
+			);
+
+			const parentId = getBlockRootClientId( clientId );
+			const parentBlockName = getBlockName( parentId );
+
+			const displayControls = hasBlockSupport(
+				parentBlockName,
+				'__experimentalDisplayChildBlockControls',
+				true
```

Comment:
nitpick: While the current name `__experimentalDisplayChildBlockControls` and default value of `true` works, I wonder if we should inverse this for better clarity:
- Rename to `__experimentalHideChildBlockControls`
- Make default `false`
Otherwise I feel it's a bit odd introducing a new block supports that declares a feature that already exists: child block controls. It might be confusing to see blocks that don't include this particular block supports at all, but still display their child's block controls.

Reply:
That reasoning makes sense to me, I agree that'd help with clarity and went ahead with the rename in https://github.com/WordPress/gutenberg/pull/36371/commits/e9ff5ced922af98655abdf7db29027d196256db0. :)

Reply:
The change looks good 👍